manifest-parser.js

```diff
@@ -224,12 +225,15 @@ var ManifestParser = (function() {
   function _parse(string) {
     // TODO: temporary while ManifestParser is a collection of static methods.
     _logs = [];
+    _tips = [];
     _success = true;
 
     try {
       _json_input = JSON.parse(string);
     } catch (e) {
       _logs.push("File isn't valid JSON: " + e);
+      _tips.push("Double quotes are considered valid JSON, single quotes aren't. Make sure you use double quotes.");
```

Comment:
I would do something like:
```js
_tips.push('Your JSON failed to parse, these are the main reasons why JSON parsing usually fails:\n - Double quotes should be used around property names and for strings. Single quotes are not valid.\n - JSON specification disallow trailing comma after the last property even if some implementations allow it.');
```


IOW:
- mention that the tips are related to the JSON parsing failure.
- mention when double quotes are usually used.
- use single quotes ;)
- optional: make this only one tip?

lgtm with this.